src/transformers/models/bit/modeling_bit.py

```diff
@@ -323,15 +343,15 @@ def extra_repr(self) -> str:
         return "p={}".format(self.drop_prob)
 
 
-def make_div(v, divisor=8):
+def make_div(value, divisor=8):
     min_value = divisor
-    new_v = max(min_value, int(v + divisor / 2) // divisor * divisor)
-    if new_v < 0.9 * v:
-        new_v += divisor
-    return new_v
+    new_value = max(min_value, int(value + divisor / 2) // divisor * divisor)
+    if new_value < 0.9 * value:
+        new_value += divisor
+    return new_value
 
 
-class BitPreActivationBottleneckLayer(nn.Module):
+class BitPreActBottleneckLayer(nn.Module):
```

Comment:
```suggestion
class BitPreActivationBottleneckLayer(nn.Module):
```
This is a clearer name

Reply:
FIxed in [5b51e42](https://github.com/NielsRogge/transformers/pull/48/commits/5b51e42b5a2423532a7fb534ec627672e53d73f3)